packages/turbo-utils/src/getTurboConfigs.ts

```diff
@@ -0,0 +1,89 @@
+import fs from "fs";
```

Comment:
Yeah, I really want "resolved config for workspace" to come from `turbo` itself so that we don't have to keep this in sync and require particular versions for particular versions of Turborepo.